README.md

```diff
@@ -501,9 +501,16 @@ Example:
 
 Yes.
 
-### Does this library work with Angular 5.x?
+### Does this library work with Angular Versions < 8.0.0?
```

Comment:
Maybe we go with a more general title, something along the lines of `Can I use this library with an older version of Angular?`